### PR TITLE
Finished install script that installs python dependencies on system

### PR DESCRIPTION
**Pull request by Gabriel-Ivarsson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Gosplat uses the FastText model trained on a large number of open-source project
 The FastText model is pre-trained and of course, will not by any means save any data of the projects it's used in. The model is downloaded from a GitLab repo and ready to be used afterwards.
 
 ## Goals of this project
-Gosplat's and our goals are to create a nice, quick and easy way to analyze possible holes in your or your team's file structure. It may give helpful hints which can aid in creating a more maintainable environment for future projects to come!
+Gosplat's and our goals are to create a nice, quick and easy way to analyze possible holes in you/your team's file structure. It may give helpful hints which can aid in creating a more maintainable environment for future projects to come!
```
Comment (NoahHakansson) [inline README.md:12]: Maybe `code structure` and not `file structure` ?

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,116 @@
+ansicolors==1.1.8
+argcomplete==2.0.0
+attrs==22.1.0
+awscli==1.27.53
+bcrypt==3.2.2
+Beaker==1.10.0
+beautifulsoup4==4.11.2
+blivet==3.5.0
+blivet-gui==2.4.0
+botocore==1.29.53
```
Comment (NoahHakansson) [inline requirements.txt:10]: Should we really have all these in requirements.txt, feel like it is all the sub dependencies of gensim etc. Only gensim and ansi-colors should be enough or?

Comment (Gabriel-Ivarsson) [reply requirements.txt:10]: They need to be installed either way and this doesn't hurt that. Better to be sure in this moment

**Review by NoahHakansson — APPROVED** [review]
Trodde jag aprrovedd sryy

**Pull request merged**